backend/app/post/controller.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any
+from uuid import UUID
+
+from flask.views import MethodView
+from flask_smorest import Blueprint, abort
+from flask_smorest.error_handler import ErrorSchema
+
+from database import TupleNotFound
+
+from .dto import CreatePostSchema, PostSchema, UpdatePostSchema
+from .service import PostService
+
+bp = Blueprint("post", __name__, url_prefix="/post", description="Operation about post")
```

Comment:
```suggestion
bp = Blueprint("post", __name__, url_prefix="/posts", description="Operation about post")
```

[RESTful API](https://martinfowler.com/articles/richardsonMaturityModel.html#level1) 스타일의 API는 기본적으로 복수형의 명사로 명명되어야 합니다.